pilot/pkg/networking/core/v1alpha3/networkfilter.go

```diff
@@ -82,6 +84,7 @@ func buildOutboundNetworkFiltersWithSingleDestination(push *model.PushContext, n
 		tcpProxy.IdleTimeout = durationpb.New(idleTimeout)
 	}
 	maybeSetHashPolicy(destinationRule, tcpProxy, subsetName)
+	tunnelingConfigBuilder(tcpProxy, destinationRule, subsetName)
```

Comment:
I see in the API its only TCP+TLS, as implemented here, but nowhere mentions why. Maybe on `tunnelingConfigBuilder` we can add a comment explaining why we skip HTTP?

Reply:
Costin agreed to add support for HTTP in a subsequent pull request, because it will be more complex change. So since I'm going to extend it soon, I didn't document why it's not supported for now.
On the other hand, `tunneling_config` is a property of the `TcpProxy` which is not used for HTTP listeners, so I don't see a need to explain why HTTP listeners don't use it.